### PR TITLE
[WIP] Smoke tests for OSP and RHV

### PR DESCRIPTION
**Pull request by sshveta**
{{pytest: cfme/tests/v2v/test_v2v_smoke.py --use-provider rhv43 --use-provider vsphere67-ims --provider-limit 2 -vvvv}}

To be used with RHV CI pipeline . 
It includes two basic tests for VDDK and SSH for RHV and OSP each.

### CODE REVIEW

File: cfme/tests/v2v/test_v2v_smoke_tests.py [hunk 1]
```diff
@@ -76,3 +80,49 @@ def _cleanup():
...
+    )
+
+    assert migration_plan.wait_for_state("Started")
+    assert migration_plan.wait_for_state("In_Progress")
+    assert migration_plan.wait_for_state("Completed")
+    assert migration_plan.wait_for_state("Successful")
+
+    # validate MAC address matches between source and target VMs
+    src_vm = mapping_data_multiple_vm_obj_single_datastore.vm_list.pop()
+    migrated_vm = get_migrated_vm_obj(src_vm, provider)
```
Comment (Yadnyawalkya) [inline cfme/tests/v2v/test_v2v_smoke_tests.py]: Required - Replace `get_migrated_vm_obj` with `from cfme.fixtures.v2v_fixtures import get_migrated_vm`

File: cfme/tests/v2v/test_v2v_smoke_tests.py [hunk 2]
```diff
@@ -76,3 +80,49 @@ def _cleanup():
...
     migrated_vm = get_migrated_vm_obj(src_vm_obj, provider)
     assert src_vm_obj.mac_address == migrated_vm.mac_address
+
+
+@pytest.mark.parametrize(
+    "v2v_provider_setup, mapping_data_multiple_vm_obj_single_datastore",
+    [["SSH", ["nfs", "nfs", [rhel7_minimal]]]],
+    indirect=True,
+)
+def test_single_vm_migration_with_ssh(
```
Comment (Yadnyawalkya) [inline cfme/tests/v2v/test_v2v_smoke_tests.py]: You can also remove entire `test_migration_plan` by adding `[["SSH", ["nfs", "nfs", [rhel7_minimal]]], ["VDDK", ["nfs", "nfs", [rhel7_minimal]]]]` here. This should cover RHV + VDDK/SSH and OSP + VDDK/SSH (just need to test OSP by executing).

File: cfme/tests/v2v/test_v2v_smoke.py [hunk 3]
```diff
@@ -24,55 +30,47 @@
...
-    return migrated_vm
-
-
 @pytest.mark.parametrize(
-    "mapping_data_vm_obj_single_datastore",
-    [
-        ["nfs", "nfs", rhel7_minimal]
-    ],
+    "v2v_provider_setup, mapping_data_multiple_vm_obj_single_datastore",
+    [["SSH", ["nfs", "nfs", [rhel7_minimal]]], ["VDDK", ["nfs", "nfs", [rhel7_minimal]]]],
```
Comment (kedark3) [inline cfme/tests/v2v/test_v2v_smoke.py]: Can we keep formatting consistent as we had earlier? 
```
[
	["SSH",
		["nfs", "nfs", [rhel7_minimal]]
	],
	["VDDK",
		["nfs", "nfs", [rhel7_minimal]]
	]
],
```

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request closed without merging**